godot-core/src/obj/script.rs

```diff
@@ -153,17 +154,28 @@ pub trait ScriptInstance: Sized {
     /// The engine may call this function if
     /// [`IScriptExtension::is_placeholder_fallback_enabled`](crate::classes::IScriptExtension::is_placeholder_fallback_enabled) is enabled.
     fn property_set_fallback(this: SiMut<Self>, name: StringName, value: &Variant) -> bool;
+
+    /// This function will be called to handle calls to [`Òbject::get_method_argument_count`](crate::classes::Object::get_method_argument_count)
+    /// and `Callable::get_argument_count`.
```

Comment:
Can you also link to the `Callable` method?

Reply:
That one is not exposed yet. `InnerCallable` already has this method but not the public `Callable`.